pkg/capr/installer/installer.go

```diff
@@ -156,22 +163,39 @@ func WindowsInstallScript(ctx context.Context, token string, envVars []corev1.En
 	if v, ok := ctx.Value(tls.InternalAPI).(bool); ok && v {
 		ca = systemtemplate.InternalCAChecksum()
 	}
+
 	if ca != "" {
-		ca = "$env:CATTLE_CA_CHECKSUM=\"" + ca + "\""
+		ca = capr.FormatWindowsEnvVar(corev1.EnvVar{
+			Name:  "CATTLE_CA_CHECKSUM",
+			Value: ca,
+		}, false)
 	}
+
+	var tokenEnvVar, cattleRoleNone string
 	if token != "" {
-		token = "$env:CATTLE_ROLE_NONE=\"true\"\n$env:CATTLE_TOKEN=\"" + token + "\""
+		tokenEnvVar = capr.FormatWindowsEnvVar(corev1.EnvVar{
+			Name:  "CATTLE_TOKEN",
+			Value: token,
+		}, false)
+		cattleRoleNone = capr.FormatWindowsEnvVar(corev1.EnvVar{
+			Name:  "CATTLE_ROLE_NONE",
+			Value: "\"true\"",
```

Comment:
Can you confirm that the value is expected to be a no-boolean value?

Reply:
Yes, this is supposed to be a string. The current implementation also sets it as a string 

https://github.com/rancher/rancher/blob/5fd3e0305fbd6bb82d45cd98ecb2a8c4ca42616a/pkg/capr/installer/installer.go#L163